src/components/common/TmBalance.vue

```diff
@@ -457,6 +479,55 @@ export default {
 }
 </script>
 <style scoped>
+select {
+  background: var(--input-bg);
+  color: var(--txt, #333);
+  border: none;
+}
+
+select option {
+  background: var(--app-bg);
+  color: var(--txt);
+  font-family: var(--sans);
+}
+
+.currency-flag {
+  width: 15px;
```

Comment:
`1rem` (in lunie) is `16px` so probably `1rem` is fine! 
```suggestion
  width: 1rem;
```